scripts/keys.coffee

```diff
@@ -10,95 +10,309 @@
 #	bot xyz has keys : sets xyz as the holder of keys
 #
 # Examples:
-#   :bot who has keys
-#   :bot i have keys
-#   :bot i dont have keys
-#	:bot who has keys
-#	:bot ravi has keys
+#   :bot who has <color> keys
+#   :bot i have <color> keys
+#   :bot i dont have <color> keys
+#	:bot who has <color> keys
+#	:bot ravi has <color> keys
 #
 # Author:
 #   Punit Dhoot (@pdhoot)
-#   Developer at SDSLabs (@sdslabs)
+#   Developer at SDSLabs (@sdslabs
+#  Edit:
+#   prakhar gupta
```

Comment:
It is a good practice to write the name in capitals

Reply:
ok